frontend/src/components/SignUpForm.js

```diff
@@ -39,11 +40,17 @@ const SignUpForm = () => {
       headers: { "Content-Type": "multipart/form-data" },
     })
     .then(function (response) {
-      console.log(response);
-      changeSubmitError('');
+      if(response.status === 200){
+        localStorage.setItem('token', response.data['idToken']);
+        localStorage.setItem('uid', response.data['localId']);
+        localStorage.setItem('refreshToken', response.data['refreshToken']);
+        changeSubmitError('');
+        history.push('/dashboard');
+      }
+      changeSubmitError("It seems like you've already signed up. Try logging in!");
```

Comment:
i left this blank so that no error message would show if login is successful.
but then again, the user would be redirected immediately after successful login.
should we remove this line altogether?

Reply:
No, we might hit this edge case where response is not 200 and it doesn't go thru the error condition. I left it there to handle that edge case.